### PR TITLE
Update for strata workshop

### PR DESCRIPTION
**Pull request by gAmUssA**

### CODE REVIEW

File: ksql-workshop/ksql-workshop.adoc [hunk 1]
```diff
@@ -278,8 +293,12 @@ Since KSQL is heavily based on SQL, you can do many of the standard SQL things y
 ----
 ksql> SELECT USER_ID, STARS, CHANNEL, MESSAGE FROM ratings WHERE STARS <3 AND CHANNEL='iOS' LIMIT 3;
 3 | 2 | iOS | your team here rocks!
-2 | 1 | iOS | worst. flight. ever. #neveragain
-15 | 2 | iOS | worst. flight. ever. #neveragain
+2 | 1 | iOS | worst.
```
Comment (rmoff) [inline ksql-workshop/ksql-workshop.adoc]: Are these line breaks correct?

File: ksql-workshop/ksql-workshop.adoc [hunk 2]
```diff
@@ -404,24 +426,29 @@ messages-per-sec:      0.33   total-messages:      1857     last-message: 6/20/1
...
 
-The derived stream that we've created is just another stream that we can interact with in KSQL as any other. If you run a `SELECT` against the stream you'll see new messages arriving based on those coming from the source `ratings` topic:
+The derived stream that we've created is just another stream that we can interact with in KSQL as any other.
+If you run a `SELECT` against the stream you'll see new messages arriving based on those coming from the source `ratings` topic:
 
 [source,sql]
 ----
 ksql> SELECT STARS, CHANNEL, MESSAGE FROM POOR_RATINGS;
-1 | iOS | worst. flight. ever. #neveragain
+1 | iOS | worst.
```
Comment (rmoff) [inline ksql-workshop/ksql-workshop.adoc]: same Q about line breaks

File: ksql-workshop/ksql-workshop.adoc (hunk 1 above)
Comment (gAmUssA) [reply ksql-workshop/ksql-workshop.adoc]: yeah, are they suppose to be there?

**Comment by gAmUssA** [comment]
@rmoff fixed line breaks. Thank you
Gotta merge :)

**Pull request merged**